packages/contracts-bedrock/package.json

```diff
@@ -15,7 +15,7 @@
   ],
   "scripts": {
     "build:forge": "forge build",
-    "build:with-metadata": "hardhat clean && FOUNDRY_PROFILE=echidna hardhat compile",
+    "build:with-metadata": "FOUNDRY_PROFILE=echidna yarn build:forge",
```

Comment:
Echidna needs the metadata to help it differentiate between the contracts in its execution environment.